packages/vue/src/composables/useUtils.ts

```diff
@@ -1,33 +1,34 @@
-import { AuthEventData } from '@aws-amplify/ui';
+import {
+  AuthEventData,
+  ValidationError,
+  CodeDeliveryDetails,
+  AuthenticatorRoute,
+  AuthenticatorServiceFacade,
+  SocialProvider,
+  AmplifyUser,
+} from '@aws-amplify/ui';
 
-export const facade = {
+export const facade: AuthenticatorServiceFacade = {
   error: '',
   hasValidationErrors: false,
   isPending: false,
-  route: '',
-  authStatus: '',
-  user: '',
-  validationErrors: [''],
-  codeDeliveryDetails: {
-    val: '',
-    DeliveryMedium: '',
-  },
-  change: (data?: AuthEventData) => null,
+  route: {} as AuthenticatorRoute,
```

Comment:
`AuthenticatorRoute` is a string union, not sure why we would use an object here

Reply:
OK, I'll remove the cast and add the route in.

Reply:
[b048bfb](https://github.com/aws-amplify/amplify-ui/pull/3469/commits/b048bfb115b769e358fb6b1374bbef8cfa03838d)